### DISCUSSION TITLE
移动 IPV6 优选 Cloudflare CDN 香港节点绕道，延迟高、丢包高、速度慢

### DISCUSSION THREAD

**Discussion by zasl** [XIU2/CloudflareSpeedTest#322] [讨论]
以前CF优选V6香港大部分在ping值7x左右，随便一个ip可以4k不卡，大概前几天突然网络很卡，找所有V6的优选地址PING值都在230左右，而且丢包很多，网络速度还被限制了，在windows的v2rayN下测速不超过1M，youtube别说4k了，720都开始卡了，并且ping时有不少的丢包，以前偶尔有一下丢包，限制是肉眼可见的丢包了。v2rayN的测速偶尔能看到瞬时速度在1M以上但是马上就下降到了0.2M/s，暂时我这里的CF套V6已经快用不了了，持续时间快一周了。

**Comment by XIU2** [comment]
正常，因为前段时间 Cloudflare 香港节点 IPv6，**移动不再直连，改绕道**了，所以你扫到的 IP 延迟相比之下都会明显增加。
**延迟高、丢包高**，速度怎么可能快，特别是丢包对速度延迟影响最大。

> 香港节点一直以来都只有 移动 是直连的，电信 联通 从古至今都是绕道的。

这属于 Cloudflare CDN 节点线路问题，找我也没用，Cloudflare 不归我管~
毕竟这种情况，每年都有（今天上半年就已经绕道过一次了），但什么时候恢复直连，我也不知道。

**Comment by zasl** [comment]
> 正常，因为前段时间 Cloudflare 香港节点 IPv6，**移动不再直连，改绕道**了，所以你扫到的 IP 延迟相比之下都会明显增加。 **延迟高、丢包高**，速度怎么可能快，特别是丢包对速度延迟影响最大。
> 
> > 香港节点一直以来都只有 移动 是直连的，电信 联通 从古至今都是绕道的。
> 
> 这属于 Cloudflare CDN 节点线路问题，找我也没用，Cloudflare 不归我管~ 毕竟这种情况，每年都有（今天上半年就已经绕道过一次了），但什么时候恢复直连，我也不知道。

使用Best Trace跟踪并没有绕道，而是直接限速，现在是全白天限速带丢包，凌晨的时候虽然高ping，但是不丢包不限速，4k没问题，只是凌晨那几个小时的时间网络是相对正常的。这个地区也是国内比较早测试dns污染的地方。

**Comment by XIU2** [comment]
那和绕道也没啥区别，我是联通，我也不清楚，是前几天在 LOC 看到有人说移动绕道了~

这种情况，要么是 Cloudflare 干的，要么是 运营商干的，无论哪种都不是你我能解决的。

****

如果你优先 IP 是用来套代理的，那么你需要知道，代理套 Cloudflare CDN 应该是**保底/备用方案**，而不应该是**唯一方案**，否则就是这结果。即使未来某一天墙完全封了 Cloudflare CDN 也不稀奇。

**Comment by zasl** [comment]
> 那和绕道也没啥区别，我是联通，我也不清楚，是前几天在 LOC 看到有人说移动绕道了~
> 
> 这种情况，要么是 Cloudflare 干的，要么是 运营商干的，无论哪种都不是你我能解决的。
> 
> 如果你优先 IP 是用来套代理的，那么你需要知道，代理套 Cloudflare CDN 应该是**保底/备用方案**，而不应该是**唯一方案**，否则就是这结果。即使未来某一天墙完全封了 Cloudflare CDN 也不稀奇。

感觉还是运营商干的，确实我们无力解决，只是我把这样的情况说一下，因为现在IPV6模式已经有很严重的限速了，如果方便，能否提供一下loc的帖子看一下吗？

**Comment by XIU2** [comment]
我只是逛论坛是随手看到的，哪还会记得是哪个帖子的。。。

不过，移动的话不稀奇，类似的网络问题移动经常干，所以放到移动身上，感觉一点也不违和、不奇怪~
